apps/web/app/pages/portfolio/portfolio.page.tsx

```diff
@@ -52,11 +72,28 @@ export function PortfolioPage() {
     return assets.sort(sortAssetsByValue);
   }, [btcQuery.data, sip10Query.data, stxQuery.data]);
 
-  const isLoading = btcQuery.isLoading || sip10Query.isLoading || stxQuery.isLoading;
+  if (status !== 'connected') {
+    return (
+      <>
+        <PortfolioPageLayout
+          dummyDataMode
+          overview={<PortfolioSummary balance={dummyTotalBalance} />}
+          assetCount={dummyPortfolioAssets.length}
+          assetList={<PortfolioTable assets={dummyPortfolioAssets} isLoading={false} />}
+          visualization={<PortfolioChart assets={dummyPortfolioAssets} />}
+          activityList={<ActivityList activity={[]} isLoading={false} />}
+        />
+        <WalletConnectionModal isOpen={true} />
+      </>
+    );
+  }
+
+  const isLoading =
+    isConnected && (btcQuery.isLoading || sip10Query.isLoading || stxQuery.isLoading);
 
   return (
     <PortfolioPageLayout
-      overview={<PortfolioSummary />}
+      overview={<PortfolioSummary balance={isConnected ? totalBalance : dummyTotalBalance} />}
```

Comment:
I don't think you need this check for `isConnected` as you are returning early above. 

Also you already have a const for `isConnected` so you can use that instead of`status !== 'connected'` on line 75?

Reply:
thanks pete yes can remove